server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -740,6 +729,39 @@ private ConnectionEntity getEntity(@NonNull Long id) {
         return repository.findById(id).orElseThrow(() -> new NotFoundException(ResourceType.ODC_CONNECTION, "id", id));
     }
 
+    private List<ConnectionConfig> entitiesToModels(@NonNull List<ConnectionEntity> entities,
+            @NonNull Long organizationId, @NonNull Boolean withEnvironment) {
+        if (CollectionUtils.isEmpty(entities)) {
+            return Collections.EMPTY_LIST;
+        }
+        Map<Long, Environment> id2Environment;
+        if (withEnvironment) {
+            id2Environment = environmentService.list(organizationId).stream()
+                    .collect(Collectors.toMap(environment -> environment.getId(), environment -> environment));
+        } else {
+            id2Environment = new HashMap<>();
+        }
+        return entities.stream().map(entity -> {
```

Comment:
Is method entityToModel still used? if not, prefer deleting it~

Reply:
yes, entityToModel still works in create/update/detail, while entitiesToModels would be used in batch operation.

Reply:
Maybe use entitiesToModels instead of entityToModel since there is a lot of code duplication in them